backend/src/models/user.js

```diff
@@ -88,8 +88,7 @@ function validateSignUp (req) {
       .keys({
         type: Joi.string().valid('Point').default('Point'),
         coordinates: Joi.array().items(Joi.number()).required()
-      })
```

Comment:
Could you unstage this change from this file, as this is not a valid/required change.

Reply:
That's done